lib/runner.rb

```diff
@@ -0,0 +1,14 @@
+require './card.rb'
+require './turn.rb'
+card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+
+
+card_2 = Card.new("Which continent is the coldest?", "Antartica", :Geography)
+
+card_3 = Card.new("Gotta catch'em all, is a phrase from what hit show originating in the mid 90's?", "Pokemon", :Popculture)
+
+
+card_4 = Card.new("The cartoon character 'Spongebob', lives in what kind of fruit?", "Pineapple", :Popculture)
+
+
+card_5 = Card.new("While the United States uses farenheit to label temperature measurements, what does majority of the world use?", "Celcius", :STEM)
```

Comment:
Careful getting further ahead than what the iterations and interaction patterns guide you through. We want you to focus on the logic before focusing on how the game will run.